pkg/status/management_conflict_error.go

```diff
@@ -113,6 +117,6 @@ func (m *managementConflictErrorImpl) Is(target error) bool {
 	return m.underlying.Is(target)
 }
 
-func (m managementConflictErrorImpl) ConflictingObjectID() core.ID {
+func (m managementConflictErrorImpl) ObjectID() core.ID {
 	return core.IDOf(m.resource)
```

Comment:
missing the pointer

Reply:
Should be fine. It doesn't mutate the object, so it doesn't need a pointer. Methods on a struct are accessible to pointers of the struct, so it still satisfies the interface.

Though, we should probably be internally consistent and either use or don't use the pointer on all the methods that don't mutate the struct.

Reply:
Added a pointer here just to be consistent with all the other methods on this struct.